subcommand/server-acl-init/command_ent_test.go

```diff
@@ -16,105 +17,100 @@ import (
 
```

Comment:
You did a great job updating these tests to pass both `-create-inject-auth-method` and `-create-inject-token` however in my opinion looking at it now, it just makes those tests too complicated when it requires an additional `t.Run()` outside of the existing `t.Run()`.

I think we can argue that we can get away with just switching to the new `-create-inject-token` for all the ent tests because we've already tested in the oss tests that the deprecated flag works to create the auth method and because the tradeoff in test complexity isn't worth it.

Reply:
I was thinking about this when I was doing it, it got ugly fast. I think we can just get one back compatible test to validate namespaces for ent and we'd be OK setting the rest to current valid flags, thoughts?

Reply:
Yeah that would work too.